easybuild/easyconfigs/i/imagecodecs/imagecodecs-2022.9.26-foss-2022a.eb

```diff
@@ -0,0 +1,72 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+# update: Thomas Hoffmann (EMBL), Denis Kristak (Inuits)
+easyblock = 'PythonBundle'
+
+name = 'imagecodecs'
+version = '2022.9.26'  # opting for older version because of compatibility issues with LibTIFF
```

Comment:
@deniskristak What kind of problems did you run into exactly here with the latest `imagecodecs` version?

Reply:
@boegel I think it was these errors: https://github.com/vscentrum/vsc-software-stack/blob/wip/76_cellpose/old_logz/easybuild-run_cmd-82n3t0f4.log
( `error: unknown type name TIFFOpenOptions` )
I didn't find much on how to solve the issue as such, but changing simply reverting to an older version seems to have worked.